app/models/manageiq/providers/redhat/infra_manager/event_parser.rb

```diff
@@ -206,4 +209,27 @@ def self.parse_new_resource_pool(cluster)
 
     rp_hash
   end
+
+  def self.hash_to_ostruct(hash)
+    OpenStruct.new(
+      hash.each_with_object({}) do |(key, val), memo|
+        memo[key] = val.kind_of?(Hash) ? hash_to_ostruct(val) : val
+      end
+    )
+  end
+
+  def self.ovirtobj_to_hash(obj)
+    obj.instance_variables.each_with_object({}) do |k, h|
+      val = obj.instance_variable_get(k)
+
+      h[k.to_s[1..-1]] = case val
```

Comment:
Just curious what does `k.to_s` look like that you had to strip out the first char?

Reply:
for the '@'

Reply:
Ah of course